classes/battle/BattleManager.php

```diff
@@ -925,7 +929,7 @@ public function jutsuCollision(
                     $collision_text .= "[player] swiftly evaded " . ($damage_reduction * 100) . "% of [opponent]'s damage!";
                 }
                 else {
-                    $collision_text .= "[player] cast [gender2] jutsu before [opponent] cast, negating " .
+                    $collision_text .= "[player] cast [attackerGender2] jutsu before [opponent] cast, negating " .
```

Comment:
```suggestion
                    $collision_text .= "[player] cast [gender2] jutsu before [opponent] cast, negating " .
```
Going along with above